viper/modules/rats/jrat.py

```diff
@@ -158,13 +169,14 @@ def parse_config(raw_config, enckey):
     config_dict["EncryptionKey"] = enckey.encode('hex')
     return config_dict
 
+
 def config(data):
     enckey, conf = get_parts(data)
-    if enckey == None:
+    if not enckey:
```

Comment:
Not quite sure what you are getting at. This change is purely syntax.. it does not change the behavior. 

Of course you can say "if `enckey == None` is exactly the same as `if not enckey` - why change it.. it's a question about style and convention.